### PR TITLE
Update whitelist.yml

### PR DESCRIPTION
**Pull request by educlerici-zondax**

### CODE REVIEW

File: whitelists/whitelist.yml [hunk 1]
```diff
@@ -1,312 +1,326 @@
...
+- name: "BESTVALIDATOR/PAS3"
   address: 5DJb44caCLAgfgxBDcP3Q5GQ4SGpMHAGyrXPyMnz6FrYSXED
-- name: address
+- name: "BESTVALIDATOR/PAS4"
   address: 5Cf7QK3jhYhX2iWdvFmQZnL2TDKkRjiNQ1DsnWfMX9SfHyJy
-- name: address
+- name: "BESTOVALIDATOR/PAS5"
   address: 5HGuNeqcxQnzyFFhGDaD2db4bt6y7diUnCDFb3X5fUfBwSgh
-- name: address
+ - name: "BESTOVALIDATOR/PAS6"
```
Comment (emmanuelm41) [inline whitelists/whitelist.yml:21]: Wrong indentation here.

File: whitelists/whitelist.yml [hunk 2]
```diff
@@ -316,15 +330,9 @@
   address: 5Fjmh8QuPkLAcErd6oAGTACvy99ifEA5W3QeCnKVfUxH3uC6
 - name: address
```
Comment (emmanuelm41) [inline whitelists/whitelist.yml:331]: There are still a few addresses without a name. Is this correct?

**Review by emmanuelm41 — COMMENTED** [review]
Left a few comments.

File: whitelists/whitelist.yml (hunk 2 above)
Comment (educlerici-zondax) [reply whitelists/whitelist.yml:331]: yes emma, i will send those tomorrow so the team can help me to identify them

File: whitelists/whitelist.yml (hunk 1 above)
Comment (educlerici-zondax) [reply whitelists/whitelist.yml]: fixed!

**Review by emmanuelm41 — APPROVED** [review]

**Pull request merged**